ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -112,6 +124,22 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
     }
   }
 
+  const hasETAGroupError = (group: MhrRegistrationHomeOwnerGroupIF): boolean => {
+    // Variable to track if group has a valid combination of Executor/Trustee/Admin (ETA) Owners
+    let hasETAGroupError = false
+    // If a group contains multiple ETA's within one group, set invalid
+    if (group.owners.filter(owner => executorTrusteeAdmin.includes(owner.partyType)).length > 1) {
+      console.log('Multiple ETA Error')
+      hasETAGroupError = true
```

Comment:
Here we could just `return true` and skip the second condition check below.

Reply:
fixed